File: SRC/CPCDOS EDITOR/Objets.vb
Public Class Objets
    Private Sub RibbonControl1_Click(sender As Object, e As EventArgs) Handles RibbonControl1.Click

    End Sub

    Private Sub Objets_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub Objets_MouseMove(sender As Object, e As MouseEventArgs) Handles Me.MouseMove
        Opacity = 1.0
    End Sub

    Private Sub ButtonX14_Click(sender As Object, e As EventArgs) Handles ButtonX14.Click
        Dim bouton As New Button()
        bouton.Location = New Point(147, 121)
        bouton.Size = New Size(90, 100)
        bouton.Visible = True
        Fenetre.Controls.Add(bouton)
    End Sub


End Class

File: SRC/CPCDOS EDITOR/Objets.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace OSMaker
{
    public partial class Objets
    {
        public Objets()
        {
            InitializeComponent();
            _ButtonX14.Name = "ButtonX14";
            _RibbonControl1.Name = "RibbonControl1";
        }

        private void RibbonControl1_Click(object sender, EventArgs e)
        {
        }

        private void Objets_Load(object sender, EventArgs e)
        {
        }

        private void Objets_MouseMove(object sender, MouseEventArgs e)
        {
            Opacity = 1.0d;
        }

        private void ButtonX14_Click(object sender, EventArgs e)
        {
            var bouton = new Button();
            bouton.Location = new Point(147, 121);
            bouton.Size = new Size(90, 100);
            bouton.Visible = true;
            My.MyProject.Forms.Fenetre.Controls.Add(bouton);
        }
    }
}